packages/manager/apps/web-domains/src/alldoms/__mocks__/serviceInfoDetail.ts

```diff
@@ -11,32 +16,86 @@ export const serviceInfoDetail: TServiceDetail = {
     offer: 'gold',
     type: ServiceInfoType.FrenchInternational,
   },
-  domainAttached: [
-    'testdomain.be',
-    'testdomain.biz',
-    'testdomain.com',
-    'testdomain.eu',
-    'testdomain.fr',
-    'testdomain.info',
-    'testdomain.net',
-    'testdomain.org',
-  ],
+  domainAttached: {
+    currentState: {
+      domains: [
+        {
+          name: 'testdomain.be',
+          registrationStatus: DomainRegistrationStateEnum.Registered,
+          expiresAt: '2025-10-10T18:48:22+02:00',
+          mainState: 'OK',
+          suspensionState: 'NOT_SUSPENDED',
+          protectionState: 'PROTECTED',
+          extension: 'be',
+          dnssecActivated: false,
+          nameServers: [
+            {
+              nameServer: 'testdns.ovh.net',
+            },
+            {
+              nameServer: 'testdns.ovh.net',
+            },
+          ],
+        },
+        {
+          name: 'testdomain.eu',
```

Comment:
```suggestion
          name: 'alldom-french-international-example.eu',
```
Please use this label everywhere